src/php/lib/Grpc/AbstractCall.php

```diff
@@ -33,7 +33,7 @@
  */
 namespace Grpc;
 
-require_once realpath(dirname(__FILE__) . '/../autoload.php');
+require_once realpath(dirname(__FILE__) . '/../../vendor/autoload.php');
```

Comment:
here and elsewhere: why is this still necessary ?
the autoload stanza in composer.json should remove the need for this.

If it's still needed, please add a comment to the autoload stanza explaining why.
If it's something that we can fix later, add an issue and link to it in the comment